File: java/Disk.java
public class Disk {

    private String type;
    private int capacity;
    private int price;

    // konstruktor
    Disk(){
    }

    Disk(String t, int c, int p){
        type = t;
        capacity = c;
        price = p;

        
    }

    // Getter and setter for atribute
    public String getType() {
        return type;
    }
    public int getPrice() {
        return price;
    }
    public void setPrice(int price) {
        this.price = price;
    }
    public int getCapacity() {
        return capacity;
    }
    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }
    public void setType(String type) {
        this.type = type;
    }

    public void showDisk() {
        System.out.println("Disk Type       : " + type);
        System.out.println("Disk Capacity   : " + capacity + " Gb");
        System.out.println("Disk Price      : Rp. " + price);
    }
    
}

File: c++/Disk.cpp
#include <iostream>
#include <string>

using namespace std;

class Disk
{
    private:
        /* private atribute */
        string type;
        int capacity;
        int price;
    public:
        Disk(){
        }

        Disk(string t, int p, int c){
            type = t;
            capacity = c;
            price = p;
        }

        //setter
        void setType(string t){
            type = t;
        }

        void setDiskCapacity(int c){
            capacity = c;
        }

        void setDiskPrice(int p){
            price = p;
        }

        //getter
        string getType(){
            return type;
        }

        int getDiskCapacity(){
            return capacity;
        }

        int getDiskPrice(){
            return price;
        }

        //print atribute
        void showDisk(){
            cout << "Disk Type       : " << type << endl;
            cout << "Disk Capacity   : " << capacity << " Gb" << endl;
            cout << "Disk Price      : Rp. " << price << endl;
        }

        ~Disk(){}
};